### ISSUE TITLE
overrideStyleFunction does not set styles

### ISSUE THREAD

**Issue by Athdemond** [MelihAltintas/vue3-openlayers#304]
**Describe the bug**
I'm trying to set a style for selected features on a map. I need to set text on top of a feature icon. But I can't just specify the icon src again, since it is dynamically specified at the rendering stage using a v-for.

```
<ol-vector-layer ref="picketsVL">
  <ol-source-vector ref="picketsVS">
    <ol-feature
      v-for="picket in pickets"
      :key="picket.properties.RecordId"
      :properties="{
              uuid: picket.properties.RecordId,
              colorId: picketsData[picket.properties.RecordId].color_id || 0,
              logicStatuses: picketsData[picket.properties.RecordId].logic_statuses || [],
            }"
    >
      <ol-geom-point
        :coordinates="Object.values(picket.geometry.coordinates[0])"
      />
      <ol-style>
        <ol-style-icon
          :src="
                  getImageSrc({
                    colorId: picketsData[picket.properties.RecordId].color_id || 0,
                    logicStatuses: picketsData[picket.properties.RecordId].logic_statuses || [],
                  })
                "
          :scale="0.7"
        />
      </ol-style>
    </ol-feature>
  </ol-source-vector>
</ol-vector-layer>
```
Therefore, I decided to rewrite the styles function and give it to the component.
```
<ol-interaction-select
  @select="featureSelected"
  :condition="selectCondition"
  :filter="selectInteractionFilter"
>
  <ol-style :overrideStyleFunction="selectedStyleFunction" />
</ol-interaction-select>

<script setup>
.........
const selectedStyleFunction = (feature) => {
  const style = new Style({
    image: new Icon({
      src: getImageSrc({
        logicStatuses: feature.get('logicStatuses'),
        colorId: feature.get('colorId'),
      }),
      scale: 0.7,
    }),
    text: new Text({
      text: '✔️',
      font: '14px sans-serif',
      zIndex: 2,
    }),
  });
  return style;
};
</script>
```

But it does not set the style, because there is an mistake in the ol-style component on 64 line. Missing return for computed property.

```
const styleFunc = computed<StyleLike>(() => {
  return (feature, resolution) => {
    if (properties.overrideStyleFunction) {
      // the next line must have a return
      properties.overrideStyleFunction(feature, style.value, resolution);
    }
    return style.value;
  };
});
```

**Comment by Athdemond** [comment]
Yes, I would

**Comment by d-koppenhagen** [comment]
fixed with #305
